website/cue/reference/components/sources.cue

```diff
@@ -86,7 +86,7 @@ components: sources: [Name=string]: {
 								enum: {
 									bytes:               "Byte frames are passed through as-is according to the underlying I/O boundaries (e.g. split between messages or stream segments)."
 									character_delimited: "Byte frames which are delimited by a chosen character."
-									length_delimited:    "Byte frames whose length is encoded in a header."
+									length_delimited:    "Byte frames whose messages are prefixed by a unsized big-endian 32-bit integer indicating the length."
```

Comment:
should this say "... prefixed by _an_ unsized big-endian"

Reply:
Hah, yes, thanks :)